### PR TITLE
Build use-cases page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build page on `/hpc/use-cases`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/d/133bou6x0uCiZxezjjr6uroq73mSPM3MdSiWpS5WA1XM/edit#)
- Some assets are missing and I left comment about them in the copy doc. Demo has been shared with the PdM for feedback.
- Demo at https://ubuntu-com-12010.demos.haus/hpc/use-cases

## Issue / Card

Fixes [#5867](https://github.com/canonical/web-squad/issues/5867)

### CODE REVIEW

**Comment by carkod** [comment]
@minkyngkm you may want to use these logos provided by @spencerbygraves 
https://github.com/canonical/ubuntu.com/pull/12005#issuecomment-1240541038

**Comment by minkyngkm** [comment]
Thanks @carkod and @spencerbygraves. The logo have been updated! :)

**Comment by spencerbygraves** [comment]
Hi @minkyngkm, we're using a different combination of type sizes to those on the page Carlos has built. He [mentioned](https://github.com/canonical/ubuntu.com/pull/12005#issuecomment-1240738707) the PM had requested something specific, perhaps it should be the same here?

image


Thanks.

**Review by carkod — APPROVED** [review]
LGTM.

Only comment is that probably "Monte Carlo" should be capitalised. I left a comment in the copy doc

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for reviewing! Yes I thought about making it `h2` for the title and `h4` for the sub title.
But Since "A new World of possibilities" is used `h2`, I just used `h3` for those title under "A new World of possibilities" What do you think? 

image

**Comment by minkyngkm** [comment]
Thanks @carkod  @spencerbygraves for reviewing! I've made some changes. Hope it looks better!

File: templates/hpc/base_hpc.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends "templates/base.html" %}
```
Comment (carkod) [inline templates/hpc/base_hpc.html:1]: Can you rebase against main branch? I created this same file but named `_base_hpc.html` (notice the underscore, to make it consistent with the other templates). Thanks

**Pull request merged**